governance/program/src/state/token_owner_record.rs

```diff
@@ -36,22 +36,25 @@ pub struct TokenOwnerRecord {
     /// This amount is the voter weight used when voting on proposals
     pub governing_token_deposit_amount: u64,
 
-    /// A single account that is allowed to operate governance with the deposited governing tokens
-    /// It can be delegated to by the governing_token_owner or current governance_delegate
-    pub governance_delegate: Option<Pubkey>,
-
     /// The number of votes cast by TokenOwner but not relinquished yet
     /// Every time a vote is cast this number is increased and it's always decreased when relinquishing a vote regardless of the vote state
     pub unrelinquished_votes_count: u32,
 
     /// The total number of votes cast by the TokenOwner
     /// If TokenOwner withdraws vote while voting is still in progress total_votes_count is decreased  and the vote doesn't count towards the total
     pub total_votes_count: u32,
+
+    /// Reserved space for future versions
+    pub reserved: u64,
```

Comment:
nit: if you want to reserve space as clearly as possible, you could declare it as `[u8; 8]`, making it totally clear that it's just buffer space.  This way, if you add in a `u16` field, it's easy to then change the reserved space to `[u8; 6]`

Reply:
good idea, done